kafka/consumerHandler_test.go

```diff
@@ -0,0 +1,116 @@
+// Package kafka provides producer and consumer to work with kafka topics
+package kafka
+
+import (
+	"encoding/json"
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	saramaMocks "github.com/Shopify/sarama/mocks"
+	"github.com/google/uuid"
+	"github.com/lvl484/positioning-filter/matcher"
+	"github.com/lvl484/positioning-filter/position"
+	"github.com/lvl484/positioning-filter/repository"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewConsumerGroupHandler(t *testing.T) {
+	db, _, err := sqlmock.New()
+	assert.Nil(t, err)
+
+	filters := repository.NewFiltersRepository(db)
+	matcher := matcher.NewMatcher(filters)
+	saramaProducer := saramaMocks.NewSyncProducer(t, nil)
+	producer := &producer{
+		KafkaProducer: saramaProducer,
+	}
```

Comment:
Kafka mock for producer is not needed, producer is an interface that can be mocked.

Reply:
In that case its really useless. Fixed